tools/locate_bins.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh -e
+# Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+# For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+have_command () {
+  eval "$1 --version >> /dev/null 2>&1"
```

Comment:
this is straight wrong! Many commands do not have either `--version` or `--help`. Commands are just programs somebody left on the path. 

One valid way to check if a command exists and should be checking which one should be in use. Example in fish:
```
[I] ⋊> ~ which cut; echo $status                                                                                                                                                                                                                       08:20:14
/usr/bin/cut
0
[I] ⋊> ~ which lololol; echo $status                                                                                                                                                                                                                   08:20:17
which: no lololol in (/home/pholat/.pyenv/bin:/home/pholat/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/var/lib/snapd/snap/bin)
1
```

Reply:
`which` is not a POSIX program and thus there is decent chance it's not installed.  You are correct that some programs do not have `--version` but all the ones being checked do have it.  This point of this is not to identify it's location but to ensure that it executes.

Reply:
I've removed this function entirely.

Reply:
@GravisZro what do you mean "is not POSIX program" can you please elaborate? And frankly can you please tell any distro that didn't have which out of the box, I've never met any :) Saying that - it's just a method that works, and `--version` checking doesn't. 
Saying that - you can check if the command exists, then it is executable, and even if its binary form matches the platform you are running it on. What's important is - what actually do you want to assert here.